### DISCUSSION TITLE
[Question] How does one set the cost of electricity to be used in the calculations?

### DISCUSSION THREAD

**Discussion by SemoTech** [teslamate-org/teslamate#1943] [General]
Apologies if this was asked before but a search revealed no answers.

How/where does one add/change the cost of electricity per kWh, so that charge calculations in teslamate are accurate?

Thank you.

**Comment by DrMichael** [comment]
You can set a geofence and then add price per kWh for that...

**Comment by SemoTech** [comment]
Got it, it was "hiding" in the geofence settings!  Thanks @DrMichael found it again!

**Reply by DrMichael** [reply]
you might want to close this then?

**Comment by SemoTech** [comment]
Happy to do that @DrMichael, but this is a "discussion" not an "issue", so no close option is relevant or available in this interface.
